woodwork/statistics_utils.py

```diff
@@ -232,3 +232,48 @@ def _get_mutual_information_dict(dataframe, num_bins=10, nrows=None):
                 )
     mutual_info.sort(key=lambda mi: mi['mutual_info'], reverse=True)
     return mutual_info
+
+
+def _get_value_counts(dataframe, ascending=False, top_n=10, dropna=False):
+    """Returns a list of dictionaries with counts for the most frequent values in each column (only
+        for columns with `category` as a standard tag).
+
+
+    Args:
+        dataframe (pd.DataFrame, dd.DataFrame, ks.DataFrame): Data from which to count values.
+        ascending (bool): Defines whether each list of values should be sorted most frequent
+            to least frequent value (False), or least frequent to most frequent value (True).
+            Defaults to False.
+
+        top_n (int): the number of top values to retrieve. Defaults to 10.
+
+        dropna (bool): determines whether to remove NaN values when finding frequency. Defaults
+            to False.
+
+    Returns:
+        top_list (list(dict)): a list of dictionaries for each categorical column with keys `count`
```

Comment:
Don't need to name the return object here `top_list (list(dict))` -> `list(dict):`